### ISSUE TITLE
Calendar widget shows the same old date and does not update

### ISSUE THREAD

**Issue by garthygarth** [plusonelabs/calendar-widget#368]
Calendar leaves old date and I have to remove one of my calendar in filter and then put it back so my widget show the correct date.

**Comment by yvolk** [comment]
Hello @garthygarth . Please check that in the widget's Settings -> Other settings -> "Lock list of events" is set to "Live data from real sources".
In a case it is not, your widget works in a "Snapshot mode", and will always show the same date and the same events.

**Comment by garthygarth** [comment]
Hi Yovlk

I cant seem to find  widget's Settings -> Other settings ->  this option on my calendar or my calendar widget app. How do I get there

**Comment by yvolk** [comment]
Lock list of events ("Snapshot mode") was added in "ToDo Agenda" v.4.0.6
![OtherSettings](https://user-images.githubusercontent.com/207716/89926195-2f089a80-dc0d-11ea-8660-b3110b8ab748.png)

**Comment by garthygarth** [comment]
This is the options I get when I go into the widget are we talking about the same widget for android phones Samsung ![Screenshot_20200814-022702_Calendar Widget](https://user-images.githubusercontent.com/69492204/90161370-5eb6ce80-ddd6-11ea-9ef4-a291aa9724c0.jpg)

**Comment by yvolk** [comment]
@garthygarth Yes, the same widget , but you use some old version. At the end of 2018 "Calendar widget" was renamed to "ToDo Agenda". See and download new version: https://github.com/andstatus/todoagenda#readme

**Comment by garthygarth** [comment]
Super thanks for that you are a rock star
